code/week6/main.cpp

```diff
@@ -0,0 +1,58 @@
+#include <opencv2/opencv.hpp>
+#include <string.h>
+#include <stdlib.h>
+
+int main() {
+    using namespace std;
```

Comment:
Why is this inside the main method? Also we should be consistent about which namespaces we include. The findBlue demo uses cv and makes std explicit, this demo does the opposite.
I recommend not including the cv namespace so it is obvious which functions are from OpenCV as opposed to standard c++.

Reply:
This comment still stands